equation_generator.py

```diff
@@ -59,6 +77,11 @@ def generate_numbers_for_division():
 
     Example: (252, 36, 7) creates "252/36=7"
     """
+    num1 = random.randint(50, 99)
+    num2 = random.randint(2, 9)
+    result = num1 * num2
```

Comment:
Unfortunately this doesn't take into account the requirement against remainders (whole-number answers only). For example, `99/9` is `11` (but is only `7` chars), but `99/8` is `12.375`.